.gitignore

```diff
@@ -1,2 +1,3 @@
 *.cabal
 .stack-work
+tags
```

Comment:
Not a huge deal, but: since its not anything in this repository, but rather some process you've set up on your own machine that would create these, I think it's more appropriate to ignore them in your own `~/.gitignore`. WDYT?

Reply:
Seems reasonable.